artifacts/Products/ShowProducts.recipes

```diff
@@ -13,35 +13,16 @@ particle ShowProduct in 'source/ShowProduct.js'
   in Product product
   consume item
 
-recipe ProductList &showShopList
+recipe MuxedProductItem &muxProduct
```

Comment:
I think this recipe (minus the `&muxProduct`) should be provided by the `ShowProduct.recipe` until such time as there is a Strategy that can create it.

Reply:
actually, i realized that we no longer need the `hostedParticle = ShowProduct` either. so the ItemMultiplexer is an absolutely generic recipe now, so it shouldn't be in a ShowProduct file at all, I think.
also I noticed that there is `ShowProduct.manifest` file, but the `ShowProduct` particle is redeclared in `ShowProducts.manifest`. How would you like to handle it?

Reply:
IMO, we can put everything in `ShowProducts.recipe` (plural) and not have `ShowProduct.recipe|manifest` at all. Sorry my first note was confusing, all I intended was for `MuxedProductItem` and `ShowProduct` to be in the same place (but it's even better not existing anywhere).